### ISSUE TITLE
Game Guru Max at standalone pc

### ISSUE THREAD

**Issue by Dieter9999** [TheGameCreators/GameGuruRepo#4958]
Hello,

i have bought GameGuru Max. It is great !
I have edit my first game and made it standalone. I copyed the standalone folder with GameGuru Max to 
another pc, but the game do not run.

What should i do ? Please help me.

The GGMax PC:

CPU: AMD FX 6350
RAM: 16 GB RAM
OS: Windows 10 Pro
HDD: Seagate Hybrid HDD 2 TB
GPU: NVIDIA GeForce GTX 1060 3 GB

Standalone PC:

Notebook Dell Latitude E7440
CPU: Intel Core i5 4310U
RAM: 8 GB RAM
HDD: 256 GB SSD
GPU: Intel HD

All games made with GameGuru run on this Notebook very vell.

**Comment by davetgc** [comment]
We'll need to know your system specs and the specs of the other PC before we can help here.

**Comment by UltraVox001** [comment]
Hello,

I would like to point out that the compatibility problem with AMD graphics cards has not been resolved, although the "amdfix.ini" file was automatically added to the game directory.

At launch everything seems to work (Menu, Sound,...), but the screen goes black as soon as you take control of the player.

**Comment by Dieter9999** [comment]
> We'll need to know your system specs and the specs of the other PC before we can help here.

Thank you for the reply.

The GGMax PC:

CPU: AMD FX 6350
RAM: 16 GB RAM
OS: Windows 10 Pro
HDD: Seagate Hybrid HDD 2 TB
GPU: NVIDIA GeForce GTX 1060 3 GB

Standalone PC:

Notebook Dell Latitude E7440
CPU: Intel Core i5 4310U
RAM: 8 GB RAM
HDD: 256 GB SSD
GPU: Intel HD 

All games made with GameGuru run on this Notebook very vell.

**Comment by Dieter9999** [comment]
Excuse mee.

OS: Windows 10 Pro.

**Comment by davetgc** [comment]
Hi @Dieter9999 - Did you mean to close this as resolved?

**Comment by davetgc** [comment]
@UltraVox001 - Could you please open a new issue regarding the AMD card, and provide your system specs.

Thanks.

**Comment by synchromesh62** [comment]
> Hello,
> 
> I would like to point out that the compatibility problem with AMD graphics cards has not been resolved, although the "amdfix.ini" file was automatically added to the game directory.
> 
> At launch everything seems to work (Menu, Sound,...), but the screen goes black as soon as you take control of the player.

Odd ... your not using an AMD Graphics cars your using a NVIDIA GeForce GTX 1060 3 GB.
you shouldnt have a problem.

**Comment by davetgc** [comment]
@synchromesh62 - The AMD issue was actually posted by another user, but on this thread.

**Comment by synchromesh62** [comment]
> @synchromesh62 - The AMD issue was actually posted by another user, but on this thread.

Ahh sorry I see it now 
Not sure why he jumped in here rather than create his own issue.
Both are different ?
